Dockerfile

```diff
@@ -10,15 +10,14 @@ WORKDIR /app
 
 COPY . .
 
-RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit && \
+RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit --ignore-scripts && \
```

Comment:
don't want to install husky during the build - this now fails loudly rather than just warning